src/Angor/Client/Pages/Settings.razor

```diff
@@ -58,6 +59,17 @@
                                 {
                                     <p class="text-danger-emphasis">Please tick the box to wipe all storage!</p>
                                 }
+                                
+                                <hr />
+
+                                <!-- Dropdown for network selection -->
+                                <div class="mb-3">
+                                    <label for="networkSelection">Select Network:</label>
+                                    <select class="form-control" id="networkSelection" @bind="selectedNetwork">
+                                        <option value="testnet" selected>Testnet</option>
```

Comment:
Could this be data driven from the network configuration class?

Reply:
It can but I think it is not urgent rn, we will for now just have main/test.